openapi/openapi/src/main/scala/endpoints/openapi/model/OpenApi.scala

```diff
@@ -3,12 +3,13 @@ package endpoints.openapi.model
 import java.io.Serializable
 
 import endpoints.Hashing
-import endpoints.algebra.Encoder
+import endpoints.algebra.{Encoder, ExternalDocumentationObject, Tag}
 
 import scala.collection.mutable
 
 /**
   * @see [[https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md]]
+  * @throws java.lang.IllegalArgumentException on creation if the tags are inconsistent with each other.
```

Comment:
```suggestion
  * @throws java.lang.IllegalArgumentException on creation if several tags have the same name but not the same other attributes.
```

Reply:
I expect that the compiler crash has something to do with the reference of the exception, though the error itself is quite unreadable. I try by restructuring this documentation a bit.